src/main/java/com/rpietraszewski/medicalclinic/model/entity/Institution.java

```diff
@@ -0,0 +1,43 @@
+package com.rpietraszewski.medicalclinic.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "INSTITUTION")
+@Entity
+public class Institution {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "INSTITUTION_ID")
+    private Long institutionId;
+    @Column(name = "NAME", unique = true)
+    private String name;
+    @Column(name = "CITY")
+    private String city;
+    @Column(name = "ZIP_CODE")
+    private String zipCode;
+    @Column(name = "STREET")
+    private String street;
+    @Column(name = "BUILDING_NUMBER")
+    private String buildingNumber;
+    @ManyToMany(mappedBy = "institutions")
+    private Set<Doctor> doctors;
```

Comment:
private Set<Doctor> doctors = new HashSet<>();
<- dzieki temu jak hibernate bedzie ci przemapowywal wiersze z bazy danych na encje to jesli nie znajdzie encji to i tak zwroci ci pusty set a nie nulla.